contrib/cirrus/latest_podman.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -eo pipefail
+
+cd "$GOPATH"/src
+
+dnf erase podman -y
+go get github.com/containers/podman
+
+cd "$GOPATH/src/github.com/containers/podman/"
+make binaries
```

Comment:
I would recommend doing a `make install PREFIX=/usr`.  This might be important as it installs configs and sets SELinux stuffs up properly.  That also puts it onto `$PATH`, so no changes to tests for `$PODMAN_BINARY` are needed.